### ISSUE TITLE
Error on install

### ISSUE THREAD

**Issue by carloschneider** [iancmyers/atom-eslint#29]
apm install eslint
No available version compatible with the installed Atom version: 1.7.4

**Comment by RezaRahmati** [comment]
Same problem for me too

**Comment by carloschneider** [comment]
@iancmyers

**Comment by meisterpeeps** [comment]
I'm getting no available version compatible with Atom version: 1.8.0

**Comment by carloschneider** [comment]
I believe the repository is abandoned. Maybe someone should create a new project.

**Comment by LadyBethJ** [comment]
@carloschneider yes, I think it too; and if you check the last commit, it was done on 31 Jul 2014, so... we should look for another linter tool

**Comment by merriam** [comment]
FYI.  The new package is linter-eslint. 
Close this?
